Fix the remaining broken links

kalbir opened the pull request:
connects to #38  

In my previous changes I changed the layout of the
pages in the application so that they would appear in a more logical
order.

In doing so I broke a bunch of relative links in the pages. This commit
fixes those links.

kalbir committed:
Fix the remaining broken links

In commits f938cc3954804f4d372f835c06bc8f024d3ec76e and
53a373c8ba4ab658b0fa0bfdcccd95a1f254a68b I changed the layout of the
pages in the application so that they would appear in a more logical
order.

In doing so I broke a bunch of relative links in the pages. This commit
fixes those links.

SteveMarshall commented on 01-getting-started/002-authenticate.md:
```diff
@@ -23,9 +23,9 @@ You must also have a valid GitHub account, be a member of the Ministry of Justic
-[**Sandbox cluster**](login.apps.cloud-platforms-sandbox.k8s.integration.dsd.io)
+* [**Non-production cluster**](login.apps.non-production.k8s.integration.dsd.io)
+* [**Test cluster**](login.apps.cloud-platforms-test.k8s.integration.dsd.io)
+* [**Sandbox cluster**](login.apps.cloud-platforms-sandbox.k8s.integration.dsd.io)
```
You’re missing the protocol on all of these links; as they are, they’ll link to `{{ site.url }}/[your_link]`, which I’m pretty sure isn’t what you mean.

SteveMarshall commented on 01-getting-started/002-authenticate.md:
```diff
@@ -65,9 +65,10 @@ $ kubectl config set-context gce --user=cluster-admin --namespace=foo \
- [Create a namespace](https://ministryofjustice.github.io/cloud-platform-user-docs/cloud-platform/env-create/#creating-a-cloud-platform-environment)
- [Deploying an application](https://ministryofjustice.github.io/cloud-platform-user-docs/cloud-platform/app-deploy/#deploying-an-application-to-the-cloud-platform)
 
+ * [Create a namespace](/01-getting-started/003-env-create/)
```
As I mentioned on Slack, most of the links that you have starting with `/` won’t work when pushed to GitHub Pages because this site ends up in a sub directory of `ministryofjustice.github.io`. Instead, you need to [pass the URL through a filter](https://jekyllrb.com/docs/github-pages/#project-page-url-structure): `{{ “/01-getting-started/003-env-create/“ | relative_url }}` or use the [`link` Jekyll tag](https://jekyllrb.com/docs/templates/#link): `{% link /01-getting-started/003-env-create.md %}`.

That way, they’ll work both locally and on the published GitHub Pages site.

SteveMarshall commented on 02-deploying-an-app/002-app-deploy.md:
```diff
@@ -49,7 +49,7 @@ Select your newly created repository from the displayed list.
 Within the repository, you will see the following section with values unique to you:
 
-![Image](images/repo-values.png)
+![Image](/images/repo-values.png)
```
The first part of this (`[Image]`) is the alt text for the image. We should either leave it empty, or add an appropriate description (ideally the latter, because I’m sure we’ll have developers join us with visual impairments).

kalbir replied:
Yeah I noticed this too, was going to do this in a separate PR as there are a few things like this to fix.

kalbir committed:
Make relative links work on GitHub

We need to include the `{{ site.baseurl }}` at the beginning of relative
links so that when we deploy to GitHub pages the relative links work --
it looks like GitHub drops in a different site base url so the relative
links don't work from the root directory.

kalbir commented:
This is getting messy, I'll open a new PR with the way you have recommended.
